### PR TITLE
fix: use unique artifacts json depending on target

### PR DESCRIPTION
**Pull request by antonbaliasnikov**
# What ❔

Fix benchmarks json.

## Why ❔

`evm` and `eravm` benchmark results JSONs were mixed up rewriting each other.

## Checklist

- [x] PR title corresponds to the body of PR.
- [x] Documentation comments have been added / updated.

### CODE REVIEW

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -172,13 +172,13 @@ jobs:
             --zkvyper './target-zkvyper/release/zkvyper' \
             --path=${{ inputs.compiler_llvm_benchmark_path || '' }} \
             --mode="${MODE}" \
-            --benchmark=${{ matrix.type }}.json
+            --benchmark=${{ matrix.type }}${{ inputs.target-machine }}.json
```
Comment (hedgar2017) [inline .github/workflows/benchmarks.yml:175]: A delimiter needed?

**Review by hedgar2017 — CHANGES_REQUESTED** [review]

File: .github/workflows/benchmarks.yml (hunk 1 above)
Comment (antonbaliasnikov) [reply .github/workflows/benchmarks.yml:175]: I was thinking about it. The thing is, the target machine can be empty, and then we'll have just an empty delimiter in the end. Adding additional logic with an optional `-` just for the internal name looked too much, so decided to go without a delimiter.

Comment (hedgar2017) [reply .github/workflows/benchmarks.yml:175]: `|| _default` maybe?
I'm thinking of making target mandatory.
Can it still be optional in this workflow?

**Review by hedgar2017 — APPROVED** [review]
Nice!

**Pull request merged**